lib/Github/Api/Notification.php

```diff
@@ -57,4 +57,19 @@ public function markRead(DateTime $since = null)
 
         $this->put('/notifications', $parameters);
     }
+    /**
+     * [UNDOCUMENTED] Gets a single notification using his ID
+     *
+     * @link https://developer.github.com/v3/activity/notifications/#view-a-single-thread
+     *
+     * @param ID|integer $id
+     */
+    public function id($id)
+    {
+        if (!is_numeric($id)) {
+            // Error
```

Comment:
It's okey if you remove this if statement. If someone uses a string they will get a 404 back.